library/patient.inc

```diff
@@ -832,7 +832,7 @@ function getPatientName($pid)
 /* find patient data by DOB */
 function getPatientDOB($DOB = "%", $given = "pid, id, lname, fname, mname", $orderby = "lname ASC, fname ASC", $limit = "all", $start = "0")
 {
-    $DOB = fixDate($DOB, $DOB);
+    $DOB = DateToYYYYMMDD($DOB);
```

Comment:
I confirmed why above is needed.

Reply:
TODO:
Although would be nice to do this processing at the originating script level. I will look into this further.

Reply:
I removed above (no fixDate or DateToYYYYMMDD) and are instead doing this from the origin scripts.